src/data/faq.json

```diff
@@ -515,7 +515,7 @@
                             "<b> Questions and answers about the Event QR code:</b>",
-                            "<ul><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'>Who generates the Event QR code?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Can the Event QR code also be set up and used for e.&nbsp;g. a private birthday party, providing this is legally possible?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'>Where can the Event QR code be generated?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Can Event QR codes from other apps be scanned with the CWA?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Is there a way to check in without a smartphone (dongle or similar)?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'>How should I scan the Event QR code?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> What happens if I scan an Event QR code but do not have the CWA installed on my smartphone?</a></li></ul>",
+                            "<ul><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'>Who generates the Event QR code?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Can the Event QR code also be set up and used for e.&nbsp;g. a private birthday party, providing this is legally possible?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'>Where can the Event QR code be generated?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Can Event QR codes from other apps (e.&nbsp;g. the 'luca' app) be scanned with the CWA?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> Is there a way to check in without a smartphone (dongle or similar)?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'>How should I scan the Event QR code?</a></li><li><a href='#check_in_qr_code' target='_self' rel='noopener noreferrer'> What happens if I scan an Event QR code but do not have the CWA installed on my smartphone?</a></li></ul>",

```

Comment:
There should be no space in English abbreviations: `e.&nbsp;g` is incorrect. 

https://de.wikipedia.org/wiki/Abk%C3%BCrzung#Schreibweise_und_Aussprache_nach_der_Rechtschreibreform_1996/2004/2006

"Wenn die Abkürzung mehrteilig ist, stehen im deutschen Sprachraum schmale Leerzeichen zwischen den Bestandteilen, z. B., d. h., i. d. R., ges. gesch., nicht jedoch im Englischen (e.g., a.m.).[7]"

Reply:
Done in https://github.com/corona-warn-app/cwa-website/commit/6526b70462a14205e6a81858dfc274ab6eb3ea5a.